london_housing/Unit_4_Challenge_Tier_3.ipynb

```diff
@@ -0,0 +1,4406 @@
+{
```

Comment:
<p>Bit of a nitpick when it comes to notebooks, but you should use snake case for variable names in python (properties_trans).   Camel case to be used for class names (class PropertiesTrans)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Kazage/Data-Science-Notebooks/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>